.gitlab/pipeline/test.yml

```diff
@@ -398,6 +398,10 @@ test-frame-ui:
   script:
     - time cargo test --locked -q --profile testnet -p frame-support-test --features=frame-feature-testing,no-metadata-docs,try-runtime,experimental
     - time cargo test --locked -q --profile testnet -p frame-support-test --features=frame-feature-testing,frame-feature-testing-2,no-metadata-docs,try-runtime,experimental
+    - time cargo test --locked -q --profile testnet -p xcm-procedural
+    - time cargo test --locked -q --profile testnet -p frame-election-provider-solution-type
+    - time cargo test --locked -q --profile testnet -p sp-api-test
+    - time cargo test --locked -q --profile testnet -p sp-runtime-interface
```

Comment:
I think you can pass the `-p` flag multiple times.

Reply:
Yeah, but this way it is better readable, IMO.